src/Language/PureScript/Errors.hs

```diff
@@ -1348,6 +1350,9 @@ prettyPrintSingleError (PPEOptions codeColor full level showDocs relPath) e = fl
     renderSimpleErrorMessage (DuplicateRoleDeclaration name) =
       line $ "Duplicate role declaration for " <> markCode (runProperName name) <> "."
 
+    renderSimpleErrorMessage (CannotUnwrapHiddenNewtypeConstructor name) =
+      line $ "The newtype constructor " <> markCode (showQualified runProperName name) <> " is not in scope, it should be imported to allow coercions to and from its representation."
```

Comment:
For consistency with the other error messages, I think there should be a sentence break and line break between ‘... is not in scope’ and ‘it should be imported...’.

Reply:
I turned “It should be imported to allow coercions to and from its representation” into its own sentence, on a new line.